validator/remote/src/main/java/tech/pegasys/teku/validator/remote/RemoteValidatorApiHandler.java

```diff
@@ -187,9 +189,10 @@ private SyncCommitteeDuties responseToSyncCommitteeDuties(final PostSyncDutiesRe
                     new SyncCommitteeDuty(
                         duty.pubkey.asBLSPublicKey(),
                         duty.validatorIndex.intValue(),
-                        duty.committeeIndices.stream()
-                            .map(UInt64::intValue)
-                            .collect(Collectors.toSet())))
+                        IntOpenHashSet.toSet(
+                            duty.committeeIndices.stream()
+                                .map(UInt64::intValue)
+                                .mapToInt(Integer::intValue))))
```

Comment:
Could just use:
```suggestion
                                .mapToInt(UInt64::intValue))))
```

Reply:
Actually I think there are probably quite a few places we could do this - worth searching for `mapToInt(Integer::intValue)` to find them I think - I only just realised it when I got to the end of reviewing. :)

Reply:
Oh yeah. That looks way better. Will update all instances.